library/core/src/commonMain/kotlin/com/attafitamim/krop/core/crop/CropperStyle.kt

```diff
@@ -47,9 +47,15 @@ interface CropperStyle {
     /** All available crop shapes */
     val shapes: List<CropShape>?
 
+    /** Force a crop shape, list of shapes will be ignored */
+    val forceShape: CropShape?
+
     /** All available aspect ratios */
     val aspects: List<AspectRatio>
 
+    /** Force an aspect ratio, other aspects will be ignored */
+    val forceAspectRatio: AspectRatio?
```

Comment:
I think instead of separate properties `forceAspectRatio` and `forceShape`, we can pass a single shape to `val shapes: List<CropShape>`, and a single aspect to `val aspects: List<AspectRatio>`, and handle this on the UI side, if we have one single element - it means it's forced, and we hide the button to select shapes and ratios. 

The `aspectLock` should still be optional through the state, even if we have a single element.

Also, I noticed that the original library doesn't handle the list of shapes and aspect ratios correctly, even if you don't pass rectangle and 16:9, it will be the default, I think we should use the first element of each list as default.

What do you think?

Reply:
That was also my first idea, but I noticed that issue as well. I'll try to fix both issues in another PR

Reply:
Do you need this PR to be merged to start a new one, or you will do it parallelly, then we compare both solutions?

Reply:
The approach is quite different, so I started on a fresh branch from the latest main [here](https://github.com/tamimattafi/krop/pull/14), this PR can be closed